### PR TITLE
Install SwiftLint via the install_contributor_tools script.

### PR DESCRIPTION
**Pull request by ajsecord**

### CODE REVIEW

**Comment by samsymons** [comment]
Good call — I was initially concerned that [this line](https://github.com/material-components/material-components-ios/blob/develop/scripts/install/install_brew#L25) might mess with the Travis run, but I haven't tested it.

**Comment by ajsecord** [comment]
I accidentally added some commits to this branch—they are now reverted.

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +15,7 @@ env:
     - DESTINATION="platform=iOS Simulator,name=iPhone 7,OS=10.2" SDK="iphonesimulator10.2"
 before_install:
   - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
-  - brew update
-  - brew install swiftlint || true  # If already installed, don't fail.
+  - scripts/install_contributor_tools
```
Comment (randallli) [inline .travis.yml:18]: this doesn't seem to match up with the PR description

Comment (ajsecord) [reply .travis.yml:18]: I'm confused. That's (part of) what this script does?

Comment (randallli) [reply .travis.yml:18]: hmm reading it again it makes sense...

**Pull request merged**